test/tests-setup.ts

```diff
@@ -0,0 +1,6 @@
+import { createUsableBatch } from './utils'
+
+export default async function testsSetup(): Promise<void> {
+  // internal env param for caching batch id
+  process.env.CACHED_BEE_BATCH_ID = await createUsableBatch()
```

Comment:
check whether the environment variable available is set already and if not then do only this instruction